src/panels/lovelace/cards/hui-picture-elements-card.js

```diff
@@ -4,6 +4,7 @@ import { PolymerElement } from '@polymer/polymer/polymer-element.js';
 import '../../../components/buttons/ha-call-service-button.js';
 import '../../../components/entity/ha-state-label-badge.js';
 import '../../../components/entity/state-badge.js';
+import '../components/hui-image.js';
```

Comment:
Please order imports by depth. This is a local Lovelace import, but it at the bottom.